CMakeLists.txt

```diff
@@ -79,6 +80,15 @@ if(ROOT_VERSION VERSION_LESS 6.14)
   set_property(TARGET ROOT::Core APPEND PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${ROOT_INCLUDE_DIRS}")
 endif()
 
+#--- optionally build with SIO -------------------------------------------------
+if(SIO_HANDLERS)
+  find_package( SIO REQUIRED)
+  if (SIO_FOUND)
+    MESSAGE( STATUS "Found SIO library - will build SIO I/O support" )
+    include_directories(${SIO_INCLUDE_DIRS})
```

Comment:
Doesn't SIO export the targets?
In any case, don't use `include_directories`, use `target_include_directories` for the applicable targets

Reply:
No, unfortunately SIO does not (yet) export its targets.